lib/vizion.js

```diff
@@ -45,8 +45,7 @@ vizion.revertTo = function(argv, cb) {
   var revision = (argv.revision) ? argv.revision : false;
   var _folder = (argv.folder != undefined) ? argv.folder : '.';
 
-  if (!revision)
-    return cb({msg: 'Cannot revert to an invalid commit revision', path: _folder});
+  if (!revision || !/^[A-Fa-f0-9]+$/.test(revision)) return cb({msg: 'Cannot revert to an invalid commit revision', path: _folder});
```

Comment:
`if (revision && /^[A-Fa-f0-9]+$/.test(revision)) return cb({msg: 'Cannot revert to an invalid commit revision', path: _folder});`

Reply:
Thank you for the tip!